src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = century
+    ? people.filter(person => person.sex === 'm')
+      .filter(men => (Math.ceil(men.died / 100) === century))
```

Comment:
```suggestion
    ? people.filter(person => person.sex === 'm' && Math.ceil(person.died / 100) === century)
```
There is no sense to call several `filter` methods separately.

Reply:
It can be done to make code more readable. Today we have seen such example on morning call.